add-ons/ansible-service-broker/ansible-service-broker.addon.remove

```diff
@@ -10,5 +10,15 @@ oc delete clusterrole asb-auth
 oc delete clusterrole access-asb-role
 oc delete broker ansible-service-broker
```

Comment:
You could update this line to something like:

```
$ oc delete -n ansible-service-broker -f https://raw.githubusercontent.com/ansibleplaybookbundle/ansible-playbook-bundle/#{APBTOOLS_REPO_TAG}/templates/openshift-permissions.template.yaml
```

This should allow you to remove everything created by the template.